manifests/version.pp

```diff
@@ -26,17 +26,27 @@
 #
 #   Default: >= 0
 #
+# [*install_gem_docs*]
+#   Default: no
```

Comment:
principle of least surprise means our default should be what we are currently doing which is installing docs.